objects/ohitbox/Step_2.gml

```diff
@@ -22,228 +22,400 @@ if (!global.game_paused)
+			{
+				//Set who the player is currently targeting
+				owner.playerOwner.target = collision_list[| i].owner.id;
+			
+				// Projectiles cannot be grabs or throws, so the code for them is ommitted here
+
+				// BLOCKING                                  Check to see if we are blocking correctly
+				if (collision_list[| i].owner.canBlock) && ((attackProperty.attackType[hitboxID] == eAttackType.LOW && collision_list[| i].owner.verticalMoveDir == -1) || attackProperty.attackType[hitboxID] == eAttackType.MID || (attackProperty.attackType[hitboxID] == eAttackType.HIGH && collision_list[| i].owner.verticalMoveDir != -1)){
+					
+					collision_list[| i].owner.prevState = eState.BLOCKING;
+					collision_list[| i].owner.state = eState.HITSTOP; // Set the player's state to hitstop
 				
-				//Allow Cancelling
-				owner.cancelable = true;
+					// Handle if the opponent is Crouch blocking or not
+					if (collision_list[| i].owner.verticalMoveDir == -1)
+					{
+						collision_list[| i].owner.sprite_index = sRussel_Crouch_Block;
+						collision_list[| i].owner.isCrouchBlocking = true;
+					}
+					else 
+					{
+						collision_list[| i].owner.sprite_index = sRussel_Block;
+						collision_list[| i].owner.isCrouchBlocking = false;
+					}
 				
-				// Depth Sorting
-				owner.depth = -1;
-				collision_list[| i].owner.depth = 0;
+					// Meter Build - P1 gets 75% meter, P2 gets 50%
+					collision_list[| i].owner.superMeter += floor(attackProperty.meterGain[hitboxID] * 0.5);
+					owner.playerOwner.superMeter += floor(attackProperty.meterGain[hitboxID] * 0.75);
 				
-				// Apply proper knockback direction to Destructable Objects
-				if (collision_list[| i].owner.isDestructibleObject)
-				{
-					collision_list[| i].owner.knockbackDirection = sign(image_xscale);
+					collision_list[| i].owner.knockbackVel = attackProperty.knockback[hitboxID];
+					owner.playerOwner.pushbackVel = attackProperty.pushback[hitboxID];
+					hasHit = true;
+					collision_list[| i].owner.blockstun = attackProperty.blockstun[hitboxID];
+					collision_list[| i].owner.shuffle = 0;
+					global.hitstop = attackProperty.attackHitstop[hitboxID];
+				
+					// Multiple hitboxes
+					ds_list_add(collision_list[| i].owner.hitByGroup, attackProperty.group[hitboxID]);
+				
+					// Depth Sorting
+					owner.depth = -1;
+					collision_list[| i].owner.depth = 0;
+					
+					// Remove 1 health from the projectile
+					owner.projectileHealth--;
+				
+					// Reset Frame Advantage Counter
+					oGameManager.frameAdvantage = 0;
+				
+					var particle = instance_create_layer(collision_list[| i].owner.x, collision_list[| i].owner.y, "Particles", oParticles);
+					with (particle) 
+					{
+						lifetime = 10;
+						sprite_index = sBlockEffect;
+						image_xscale = collision_list[| i].owner.image_xscale;
+					}
 				}
+				//Hitting	
+			else if attackProperty.attackType[hitboxID] != eAttackType.GRAB { 

```

Comment:
The { needs to go on it's own line. Also the condition of a conditional should be enclosed in parentheses.